### DISCUSSION TITLE
Ασκήσεις γραμμής εντολών

### DISCUSSION THREAD

**Discussion by p15bako** [courses-ionio/hci#1537] [Announcements]
Σχετικά με τον πίνακα των παραδοτέων και τα κριτήρια βαθμολογίας, υπάρχει αναντιστοιχία στον αριθμό των ασκήσεων γραμμής εντολών. Πιο συγκεκριμένα, στο πινακάκι έχετε 6, ενώ στα κριτήρια βαθμολογίας αναφέρονται μόλις δύο. Ποιο από τα δύο ακολουθούμε;

**Comment by Sola1r3** [comment]
Οι πρώτες εβδομάδες και τα πρώτα παραδοτέα αυτής της εργασίας μέχρι την 6η εβδομάδα είναι αναφορές με τις ενέργειες που έγιναν για την εγκατάσταση και προσαρμογή του νέου λειτουργικού συστήματος, π.χ., την 1η εβδομάδα θα πρέπει να εντοπίσετε τα χαρακτηριστικά που έχει το υλικό σας (δίσκος, μνήμη, επεξεργαστής, δίκτυο, γραφικά, κτλ)

Λογικά οι 2 ασκήσεις γίνονται μετά την έκτη εβδομάδα και τις πρώτες έξι είναι αυτό που γράφει παραπάνω.

**Reply by p15bako** [reply]
Δηλαδή οι πρώτες 2 εβδομάδες των ασκήσεων γραμμής εντολών πρέπει να είναι warmup ασκήσεις; Επίσης μετά την 6η εβδομάδα, ο πίνακας έχει 3 ασκήσεις γραμμής εντολών.

**Comment by p15bako** [comment]
Επιπλέον, υπάρχουν και διαθέσιμες ασκήσεις προγραμματισμού με κωδικό HCI, δεν υπάρχουν όμως στον πίνακα. Αυτές είναι διαθέσιμες;
